docs/cli/buttons_install.md

```
@@ -5,56 +5,49 @@ description: "CLI reference for buttons install"
 **Examples:**
 
 ```bash
-BUTTONS_REGISTRY_URL=https://registry.example buttons install @your-desk/hello
-buttons install deploy --source ../button-source
-buttons install deploy@1.2.0 --source ../button-source
+buttons install
+buttons add @your-desk/hello
+buttons add @your-desk/hello@1.2.3
 ```
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)